ga4gh/protocol.py

```diff
@@ -25,30 +58,65 @@ def convertDatetime(t):
     millis = delta.total_seconds() * 1000
     return int(millis)
 
+def getValueFromValue(value):
```

Comment:
This is a bit mysterious! What does this do? We should have docstring comments for all these public functions here.